wave_vision/include/wave/vision/tracker/tracker.hpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file
+ * Brute force matcher implementation, derived from descriptor matcher base.
+ * @ingroup vision
+ */
+#ifndef WAVE_VISION_TRACKER_HPP
+#define WAVE_VISION_TRACKER_HPP
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <opencv2/opencv.hpp>
+
+#include "wave/containers/landmark_measurement.hpp"
+#include "wave/containers/landmark_measurement_container.hpp"
+#include "wave/utils/utils.hpp"
+#include "wave/vision/utils.hpp"
+
+namespace wave {
+/** @addtogroup vision
+ *  @{ */
+
+class FeatureTrack {
```

Comment:
Okay, I have added the image number to the LandmarkMeasurements. The first and last images can be directly extracted from the vector of feature tracks pretty easily.